server/api/users/users.test.ts

```diff
@@ -24,16 +26,25 @@ describe('auth router', () => {
 
       const res = await server.exec.get(`/api/users/${user.id}`);
       expect(res.status).toBe(200);
-      expect(res.body).toEqual({
-        id: user.id,
-        discord_user_id: user.discord_user_id,
-        discord_name: user.discord_name,
-        bio: user.bio,
-        twitter_username: user.twitter_username,
-        linkedin_url: user.linkedin_url,
-        github_username: user.github_username,
-        website: user.website,
-      });
+      expect(res.body).toEqual(getExpectedUserObject(user));
+    });
+  });
+
+  describe('GET /', () => {
+    it('expects empty array if no users exist', async () => {
+      const res = await server.exec.get('/api/users/');
+      expect(res.body).toStrictEqual([]);
+    });
+
+    it('returns all users', async () => {
+      const userOne = await createUser();
+      const userTwo = await createUser();
+
+
```

Comment:
nit: New linting rules from #68 should prevent double empty lines, pull in the latest changes from `main` and re-run the linter.